test/langtools/jdk/jshell/CompletionSuggestionTest.java

```diff
@@ -773,4 +773,15 @@ public void testBrokenClassFile2() throws IOException {
         assertEval("import p.*;");
         assertCompletion("Broke|", "BrokenA", "BrokenC");
     }
+
+    public void testStatements() {
+        assertEval("String s = \"\";");
+        assertCompletion("if (s.conta|", (Boolean) null, "contains(");
```

Comment:
The `if (s.ch|` would return only one result, `chars`:

[chars](https://docs.oracle.com/en/java/javase/18/docs/api/java.base/java/lang/String.html#chars())()

How about including that to show that `char` (as mentioned in the description) will not be included?